WebGPU - Add secure context info (#26268)
mdn/content: files/en-us/web/api/webgpu_api/index.md

- {{domxref("GPUValidationError")}}
  - : One of the types of errors surfaced by {{domxref("GPUDevice.popErrorScope")}} and the {{domxref("GPUDevice")}} {{domxref("GPUDevice.uncapturederror_event", "uncapturederror")}} event. Describes an application error indicating that an operation did not pass the WebGPU API's validation constraints.

## Security requirements

> - Access to the whole APIis available only in a [secure context](/en-US/docs/Web/Security/Secure_Contexts).

Reviewer: ```suggestion
The whole API is available only in a [secure context](/en-US/docs/Web/Security/Secure_Contexts).
```
